burr/core/persistence.py

```diff
@@ -140,15 +140,17 @@ def save(
 class SQLLitePersister(BaseStatePersister):
     """Class for SQLLite persistence of state. This is a simple implementation."""
 
-    def __init__(self, db_path: str, table_name: str = "burr_state"):
+    def __init__(self, db_path: str, table_name: str = "burr_state", serde_kwargs: dict = None):
```

Comment:
Hmm, a little worried about global kwargs... Won't this have the potential for clashing?

Reply:
no, because each entry in `serde_kwargs` would be namespaced by that serialization implementation. `pandas_kwargs` vs `pydantic_kwargs` etc.